static/javascripts/components/survey_card.jsx

```diff
@@ -79,43 +90,46 @@ var SurveysPage = React.createClass({
     option filed in a question, then we post a final json to the api*/
 
     handleSubmit: function(){
-        var questions = this.state.questions;
-        var questionsLength = questions.length;
-        var optionsLength;
-        var finalOptions = [];
-        for(var i = questionsLength-1; i >=0; i--){
-            delete questions[i].key;
-            finalOptions = [];
-            optionsLength = questions[i].options.length;
-            for(var i2 = optionsLength-1; i2 >= 0; i2--){
-                finalOptions.push(questions[i].options[i2].title)
-            }
-            questions[i].options = finalOptions;
-        }
+        
         var surveyObj = {
             item_id : this.state.item_id,
-            item_type : this.state.item_type,
-            item_name : this.state.item_name,
-            questions : questions
+            item_type: this.state.item_type,
+            item_name: this.state.item_name,
+            questions: this.state.questions
         };
```

Comment:
I initially set questions to just be this.state.questions for testing purposes to make sure they're not empty, and then once I know they're not I take the keys off because otherwise React freaks out. I pretty much just moved the brackets upwards.